hedera-node/src/main/java/com/hedera/services/store/contracts/ContractsStore.java

```diff
@@ -86,6 +104,30 @@ public void prepareAccountCreation(AccountID sponsor, AccountID target, HederaAc
 		provisionalAccountCreations.put(target, new AbstractMap.SimpleEntry<>(sponsor, customizer));
 	}
 
+	public com.hedera.services.store.models.Account loadContract(Id id) {
```

Comment:
I think that it will be better if the name is contract instead of account - merkleAccount, account should be contracts not accounts